### ISSUE TITLE
Alternative resources

### ISSUE THREAD

**Issue by mkrupczak3** [mkrupczak3/machine-learning-for-software-engineers#1]
Compare with resources as available here on Kaggle:

https://www.kaggle.com/getting-started/51078#290875

**Comment by mkrupczak3** [comment]
Other Good resources:

[ClaoudML](https://www.claoudml.com/)

[Going into Data Science](https://www.linkedin.com/feed/update/urn:li:activity:6621807224446689282/)

[Towards Data Science](https://towardsdatascience.com/?gi=3a56c568e06c)

**Comment by mkrupczak3** [comment]
[learn sql using excel as a bridge](https://towardsdatascience.com/learn-sql-by-using-excel-as-a-bridge-30cea4d314e0?gi=9e351b16a4ca)

**Comment by mkrupczak3** [comment]
Randy Lao

Here's an ultimate data science starter kit:

1. Foundational Skills
• Intro to Python
https://lnkd.in/grCsv8v
• Intro to R 
https://lnkd.in/gKFiDZn
• Data Wrangling Pydata (90min)
https://lnkd.in/gEhF3-W
• EDA (20min video) 
https://lnkd.in/gT8_RKh
• Stats/Prob (Khan Academy)
https://lnkd.in/gsyGpVu

2. Technical Skills
• Data Gathering- Why API Medium 
https://lnkd.in/gvahtsN
• Intro to SQL
https://lnkd.in/giWs-3N
• Complete SQL Bootcamp
https://lnkd.in/gsgf_fF
• Data Visualization Medium 
https://lnkd.in/g3FSRgY
• Machine Learning A-Z
https://lnkd.in/gXqdBsQ

3. Business Skills
• Communication - Data Storytelling 
https://lnkd.in/gtiCSNT
• Business Analytics- Geckoboard 
https://lnkd.in/g2X-Xtp

4. Extra Skills
• NLP - How to solve 90% of NLP 
https://lnkd.in/gh8bKe4
• Recommendation Systems - Spotify
https://lnkd.in/gH2GQKu
• Time Series Analysis - Complete Guide 
https://lnkd.in/gFZU2Rb

5. Practice
• Projects/Competitions - Kaggle Kernels 
https://www.kaggle.com/
• Problem Solving Challenges - HackerRank 
https://lnkd.in/g9Ps2cb

- - - 
Hope these resources help!

👉If you want more free DS/ML resources, feel free to check out my site: www.ClaoudML.com

**Comment by mkrupczak3** [comment]
Top 10 Websites for Data Science

1. Coursera
2. EdX
3. Datacamp
4. Udemy
5. Udacity
6. Khan Academy
7. Kaggle 
8. R-bloggers
9. Analytics Vidya
10. KDNuggets

Top 10 Skills for Data Science

1. Probability & Statistics 
2. Linear Algebra
3. Python
4. R
5. SQL/Presto
6. Tableau/PowerBI
7. AWS/Azure
8. Spark
9. Excel
10. DevOps

Top 10 Algorithms for Data Science

1. Linear Regression
2. Logistics Regression
3. K-means Clustering
4. PCA
5. Support Vector Machine
6. Decision Tree
7. Random Forrest
8. Gradient Boosting Machine
9. XGboost
10. Artificial Neural Networks

Top 10 Industries for Data Science

1. Technology 
2. Finance
3. Retail
4. Telecom
5. Healthcare & Pharma
6. Manufacturing 
7. Automotive
8. Cybersecurity 
9. Energy
10. Utilities

**Comment by mkrupczak3** [comment]
[pytorch for deep learning a quick quide for starters](https://towardsdatascience.com/pytorch-for-deep-learning-a-quick-guide-for-starters-5b60d2dbb564)

[toe in the water examining deep learning and data science](https://matthew.krupczak.org/2019/11/29/toe-in-the-water-examining-deep-learning-and-data-science/)

[which gpu for deep learning (go fast)](https://timdettmers.com/2019/04/03/which-gpu-for-deep-learning/)

[Apex NVIDIA acceleration extensions for pytorch (go really fast)](https://github.com/NVIDIA/apex)
